packages/math-input/package.json

```diff
@@ -30,7 +30,8 @@
             "require": "./dist/strings.js",
             "types": "./dist/strings.d.ts",
             "source": "./src/strings.ts"
-        }
+        },
+        "./index.css": "./dist/index.css"
```

Comment:
Is index.css a good name? `perseus.css` may be better?

Reply:
`index.css` is a [default filename](https://github.com/Anidetrix/rollup-plugin-styles?tab=readme-ov-file#css-extraction) (although it's not documented explicitly there) provided by the `rollup-plugin-styles` plugin we use. 

https://github.com/Khan/perseus/blob/3d7e3b6854cf9f3a945e86988c6bbcb2b6c9e2cb/config/build/rollup.config.js#L216

I can change it, but it would take a bit more effort as we use a single rollup build file to generate Rollup build configs for each package. I'll see how difficult that is (basically, I think we'd have the `perseus` package have `perseus.css`, `perseus-editor` have `perseus-editor.css`.  More customization than that I don't think is worth it. :)

Reply:
To clarify, leave the file as `./dist/index.css`, but export it as `./perseus.css` - having an export named "index" doesn't really make sense? (or export it as `./styles.css` - something that provides a context clude)

Reply:
I think I like `./styles.css`. That gives a consistent naming across all Perseus packages, but signals what it is.